QA.md

```diff
@@ -0,0 +1,127 @@
+**OS version:** (eg. `Catalina 10.15.6` or `Ubuntu Xenial Xerus 16.04.1 LTS`)\
+**Link to build:** (eg. https://buildkite.com/monadic/radicle-upstream/builds/4727)
+
+
+### Prerequisites
+
+- [ ] download the binary package from the location provided above
+- [ ] remove any old configuration and user data
+  - on macOS: mount the `.dmg` and install Upstream in the default location
+    `/Applications`, then run:
+    ```
+    /Applications/radicle-upstream.app/Contents/MacOS/radicle-upstream
+    --reset-state
+    ```
+  - on Linux: run the `.AppImage` like so:
+    ```
+    /<PATH_TO_DOWNLOAD>/radicle-upstream-0.0.14.AppImage --reset-state
+    ```
+- [ ] confirm that the state reset was actually performed, the output should
+      look like this:
+
+  ```
+  INFO  api > Resetting application state...
+  INFO  api > done
+  INFO  api > Resetting CoCo state...
+  INFO  api > done
+  ```
+
+
+### Packaging and distribution
+
+- [ ] Check that unreleased features are not visible in the UI
+  - [ ] Issues, Revisions, Wallets, Design Sytem Guide, and anything else
+        that's in the UI but clearly not wired up
+- [ ] App icon is shown correctly
+  - [ ] macOS: dock, <kbd>⌘</kbd> + <kbd>tab</kbd> task switcher, mounted dmg,
+        app icon, "About radicle-upstream" window
+  - [ ] Linux: dock, menu bar
+- [ ] Version in the Settings screen matches version in package filename
+
+
+### Onboarding
+
+- [ ] Always shown when no identity is set up
+  - [ ] First app start
+  - [ ] Subsequent app starts if the identity creation was not completed
+  - [ ] Can't exit onboarding before identity is created and global keyboard
+        shortcuts are disabled
+- [ ] It's possible to go to the next screen by pressing <kbd>enter</kbd> if
+      the input field validations allow it
+- [ ] Handle and passphrase validations work
```

Comment:
Might I suggest a link to validation examples, perhaps?

Reply:
For posterity -- all @FintanH comments will be addressed in: https://github.com/radicle-dev/radicle-upstream/pull/941.